flake.nix

```
@@ -7,6 +7,11 @@
 
   outputs = { self, nixpkgs-lib, ... }: {
     lib = import ./lib.nix { inherit (nixpkgs-lib) lib; };
+    flakeModules = {
+      flakeModules = ./modules/flakeModules.nix;
+      default = self.flakeModules.flakeModules;
+    };
+    flakeModule = self.flakeModules.default;
```

```suggestion
      flakeModules = ./modules/flakeModules.nix;
    };
```

I think the default is ambiguous in this case: should it be the core module(s) or the `flakeModules` module? Let's keep it simple.